docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md

```diff
@@ -59,7 +59,7 @@ xsd {file.dll | file.exe} [-outputdir:directory] [/type:typename [...]][/paramet
 |**/e\[lement\]:**_element_|Specifies the element in the schema to generate code for. By default all elements are typed. You can specify this argument more than once.|
 |**/enableDataBinding**|Implements the <xref:System.ComponentModel.INotifyPropertyChanged> interface on all generated types to enable data binding. The short form is `/edb`.|
 |**/enableLinqDataSet**|(Short form: `/eld`.) Specifies that the generated DataSet can be queried against using LINQ to DataSet. This option is used when the /dataset option is also specified. For more information, see [LINQ to DataSet Overview](../../framework/data/adonet/linq-to-dataset-overview.md) and [Querying Typed DataSets](../../framework/data/adonet/querying-typed-datasets.md). For general information about using LINQ, see [Language-Integrated Query (LINQ) - C#](../../csharp/programming-guide/concepts/linq/index.md) or [Language-Integrated Query (LINQ) - Visual Basic](../../visual-basic/programming-guide/concepts/linq/index.md).|
-|**/f\[ields\]**|Generates fields instead of properties. By default, properties are generated.|
+|**/f\[ields\]**|Generates fields only. By default, get / set properties are generated in addition to the backing fields.|
```

Comment:
`get` and `set` are methods, not properties. Tweaked wording and added a backlink.

```suggestion
|**/f\[ields\]**|Generates fields only. By default, [properties with backing fields](https://docs.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/properties#properties-with-backing-fields).|
```